core/src/franchise/player/player.controller.ts

```diff
@@ -36,122 +38,127 @@ export class PlayerController {
 
     @Get("accept-rankdown/:token")
     async acceptRankdown(@Param("token") token: string): Promise<string> {
-        const payload = this.jwtService.decode(token) as RankdownPayload | null;
-        if (!payload) return "FAILED TO DECODE PAYLOAD";
+        try {
+            const payload = this.jwtService.verify<RankdownPayload>(token);
+            if (!payload) return "FAILED TO DECODE PAYLOAD";
```

Comment:
We should verify that the player in the rankdown payload matches the user's auth JWT, right?

Reply:
Tokens are validated with our jwt secret. In theory (if they had the secret) they could rank someone else down, but there's likely nothing to do to stop that since it's a simple `GET /:token` -> rankdown, not using auth.

Reply:
Gotcha, didn't realize it was an actual JWT 👍🏼